pkg/crdconversion/config_meshconfig_conversion.go

```diff
@@ -7,6 +7,12 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
+const (
+	strConfigv1alpha1 = "config.openservicemesh.io/v1alpha1"
+	strConfigv1alpha2 = "config.openservicemesh.io/v1alpha2"
+	strConfigv1alpha3 = "config.openservicemesh.io/v1alpha3"
```

Comment:
the `str` prefix in the naming is unnecessary. Let's drop these in favor of using `<pkg>.SchemeGroupVersion.String()`